pandas/io/pytables.py

```diff
@@ -1816,10 +1816,24 @@ class GenericIndexCol(IndexCol):
     def is_indexed(self):
         return False
 
-    def convert(self, values, nan_rep, encoding, errors):
-        """ set the values from this selection: take = take ownership """
+    def convert(self, values, nan_rep, encoding, errors, start=None, stop=None):
+        """ set the values from this selection: take = take ownership
+
+        Parameters
+        ----------
+
+        start : int, optional
+            Table row number: the start of the sub-selection.
+        stop : int, optional
+            Table row number: the end of the sub-selection. Values larger than
+            the underlying table's row count are normalized to that.
```

Comment:
At least here I understand the meaning of `start` and `stop` and have tried to document it.

Reply:
right, can you add all the parameters. See if you can put something down for them, some doc-string is better than none.

Reply:
:+1: 

Adding statements for `values`, `nan_rep`, `encoding`, and `errors`. 

The biggest challenge for me still is that I do not understand the general purpose of the convert() method and its arguments.